src/ripple/ledger/impl/View.cpp

```diff
@@ -1216,35 +1021,30 @@ trustDelete (ApplyView& view,
                  beast::Journal j)
 {
     // Detect legacy dirs.
-    bool        bLowNode    = sleRippleState->isFieldPresent (sfLowNode);
-    bool        bHighNode   = sleRippleState->isFieldPresent (sfHighNode);
     std::uint64_t uLowNode    = sleRippleState->getFieldU64 (sfLowNode);
     std::uint64_t uHighNode   = sleRippleState->getFieldU64 (sfHighNode);
     TER         terResult;
```

Comment:
It looks like this never gets set. Should we return `tesSUCCESS` at the bottom?

Reply:
Yep, that appears to be the case and I agree with just always returning `tesSUCCESS` at the bottom as we return `tefBAD_LEDGER` if we failed to remove the owner directories.